Frontend/Camera2Basic/CompareSizesByArea.cs

```diff
@@ -0,0 +1,17 @@
+using Android.Util;
+using Java.Lang;
+using Java.Util;
+
+namespace Camera2Basic
+{
+    public class CompareSizesByArea : Java.Lang.Object, IComparator
```

Comment:
class name shouldn't be a verb, also imo it's too specific (what if you will want to compare sizes not only by area? I think something like "SizeComparer" would fit the idea much better